pubsub/pubsub-pipe-image/pubsub-to-bigquery.py

```diff
@@ -121,7 +121,9 @@ def write_to_bq(pubsub, sub_name, bigquery):
 
 
 if __name__ == '__main__':
-    sub_name = "tweets-%s" % PROJECT_ID
+    topic_info = PUBSUB_TOPIC.split('/')
+    topic_name = topic_info[-1]
+    sub_name = "tweets%s_%s" % (topic_name, PROJECT_ID)
```

Comment:
I actually did that previously, but I think the '/' in the full path was causing issues... I could be wrong though.